### DISCUSSION TITLE
Taskbar style Windows 10 (Explorerpatcher)

### DISCUSSION THREAD

**Discussion by Hottihotte** [valinet/ExplorerPatcher#4063] [Q&A]
I use Windows and Explorerpatcher in German, how can I fix this "blemish" that everything is in German?
![1](https://github.com/user-attachments/assets/0b5aedcc-b4c5-4768-a69e-0eca9c68d2a6)

![2](https://github.com/user-attachments/assets/73a4d563-aed5-4932-b6c0-4dbffd2ca92b)
I mean the mixture of German and English, so that everything is in German.

**Comment by pyrates999** [comment]
duplicate of: https://github.com/valinet/ExplorerPatcher/issues/3519

please close this.

**Comment by milestprower92** [comment]
[Supported languages on ExplorerPatcher's taskbar implementation](https://github.com/valinet/ExplorerPatcher/wiki/ExplorerPatcher's-taskbar-implementation#languages)
